pyresample/spherical.py

```diff
@@ -366,19 +366,19 @@ def __init__(self, start, end):
     def __eq__(self, other):
         """Check equality."""
         if self.start == other.start and self.end == other.end:
-            return 1
-        return 0
+            return True
+        return False
 
     def __ne__(self, other):
         """Check not equal comparison."""
         return not self.__eq__(other)
 
     def __str__(self):
-        """Get simplified representation."""
+        """Get simplified str representation."""
```

Comment:
I'm not sure these changes were needed, but if we're going to say string lets just say string (not str).

```suggestion
        """Get simplified string representation."""
```